docs/sources/docker-hub/official_repos.md

```diff
@@ -1,189 +1,117 @@
+preferrable to have upstream software authors maintaining their corresponding
+Official Repositories, this is not a strict requirement.  This process takes
+place out in the open on GitHub, where participation is encouraged.  Anyone can
+provide feedback, contribute code, suggest process changes, or even propose a
+new Official Repository.
+
+## Should I use Official Repositories?
+
+New Docker users are encouraged to utilize Official Repos in their projects
+because they have clear documentation, promote best practices, and are designed
+for the most common use cases.  Advanced users are encouraged to review
+Official Repos as part of their Dockerfile learning process.
+Advanced users with policy constraints preventing them from using
+Official Repositories can still benefit from reviewing how they are
+constructed.
+
+A common rationale for diverging from Official Repos is to optimize for image
+size.  For instance, many of the programming language stack images contain a
+complete build toolchain and common C libraries/headers required to support
+installation of popular modules that depend on optimized code.  An advanced
+user could build a custom image with just the necessary pre-compiled libraries
+to save space and potentially reduce attack vectors in production.  In fact, a
+number of language stacks such as
+[`python`](https://registry.hub.docker.com/_/python/) and
+[`ruby`](https://registry.hub.docker.com/_/ruby/) have `-slim` tag variants
+designed to fill this need.  Even when these are not sufficient, it is still
+recommended to inherit from an Official Repository base OS image to leverage
+the ongoing maintenance work, rather than duplicating these efforts.
+
+## How can I get involved?
+
+All Official Repositories contain a User Feedback section in their
+documentation which covers the details for that specific repository.  In most
+cases, the GitHub repository which contains the Dockerfiles for an Official
+Repo will also have an active issue tracker.  General feedback and support
+questions should be directed to `#docker-library` on Freenode IRC.
+
+## How do I create a new Official Repository?
+
+While this page contains a high level overview of the process, detailed
+objective requirements are covered in the following GitHub repositories that
+should be carefully reviewed before submitting a proposal:
+
+* [docker-library/official-images](https://github.com/docker-library/official-images)
+
+* [docker-library/docs](https://github.com/docker-library/docs)
+
+An Official Repository starts out as a proposal in the form of a Pull Request
+to
+[docker-library/official-images](https://github.com/docker-library/official-images).
+The Official Repos team, with help from community contributors, formally review
+the proposal and provide feedback to the author.  This review process usually
+requires a bit of back and forth to resolve, but will ultimately yield a merged
+Pull Request, or a rejection.
+
```

Comment:
Only one comment: perhaps a sentence or two here about `docs`; both are mentioned above, but only `official-images` is explained.

!m @psftw :smile: